lua/dotfyle_metadata/keymaps.lua

```diff
@@ -1,32 +1,51 @@
 ---Get all user defined keymaps
 ---@return table
+
+local function check_map(keymap)
+  -- check if the mapping is one we want to include
+
+  if  keymap.mode ~= "n" and keymap.mode ~= "i"
+  and keymap.mode ~= "x" and keymap.mode ~= "v"
```

Comment:
Only need to check for `n`, `v` and `i`.

Reply:
If we omit "x" then I get none of my visual mappings.
Also if you're using "v" there's a pretty high chance you should change that to "x"

Reply:
It's very rare to see users use `x` mapping and use `v` mappings instead. Got it!